data/operational.csv-metadata.json

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
Is this removal intentional?

Reply:
Yes – my suggestion is to have two CSV sources. I think the third file to store date created & date modified introduces unnecessary complexity. 

Also, using the URL for the primary identifier doesn't work for all records - how do we provide dates for catalogue entries without URLs?

Reply:
Agree that the URL as primary doesn't work at the moment. Options need to be considered though, e.g. requiring either the URL or the URL SHA256 if there is an issue with making it public. Having it optional is an option I'd like to avoid unless the previous idea is deemed unfeasible.

In regards to the split, then it means that the sources will be like they are now but the _published_ data will be different. I expect the CSVW to be one way to consume the catalogue.

Reply:
Based on our conversation this morning, I think this is no longer relevant for this PR